packages/enhanced-img/types/index.d.ts

```diff
@@ -1,11 +1,14 @@
+import 'svelte/elements';
```

Comment:
I'm not much of a typescript expert. why was this changed from `import { HTMLImgAttributes } from 'svelte/elements';`?

Reply:
Since `HTMLImgAttributes` is not used anywhere else outside the declare block, typescript compains that it is an unused import. (similar to https://github.com/microsoft/TypeScript/issues/45061)

So one way to solve this is to import the module itself. Other option is to declare a type alias outside the `declare` block. Third option is to use the dynamic type import syntax.